nexus/post.ts

```diff
@@ -348,6 +348,23 @@ const PostMutations = extendType({
           },
         })
 
+        // automatically subscribe the post author to all PostComments made on their post
```

Comment:
I think we can do this as a nested write when we create the post and save a round trip to the database

Reply:
Done!